### PR TITLE
 Display completeness fields if object is not complete and raise an error if needed

### PR DESCRIPTION
**Pull request by numahell**
Display completeness fields in detail view if object is not complete, for models with PublishableMixin:

- Trek, POI
- TouristicContent, TouristicEvent
- Dive
- Site, Course

With `COMPLETENESS_LEVEL = 'error_on_publication'`, form raises a validation error on publish, and with  or `COMPLETENESS_LEVEL = 'error_on_review'` it also raises error on review.

### CODE REVIEW

File: geotrek/common/forms.py [hunk 1]
```diff
@@ -160,6 +184,65 @@ def check_structure(self, obj, structure, name):
...
+    @property
+    def published_languages(self):
+        """Returns languages in which the form has published data.
+        """
+        languages = [language[0] for language in settings.MAPENTITY_CONFIG['TRANSLATED_LANGUAGES']]
+        if settings.PUBLISHED_BY_LANG:
+            return [language for language in languages if self.cleaned_data.get(f'published_{language}', None)]
+        elif self.any_published:
+            return languages
+        return []
```
Comment (submarcos) [inline geotrek/common/forms.py:202]: not covered by tests

**Comment by submarcos** [comment]
add a changelog entry please

File: geotrek/common/templates/common/publishable_completeness_fragment.html [hunk 2]
```diff
@@ -9,7 +9,11 @@
 {% else %}
     {% if not object.is_publishable %}
         <div class="alert alert-danger"><strong>{% trans "Object is published but" %}</strong>
-        {% if not object.is_complete %}{% trans "looks incomplete;" %}{% endif %}
+        {% if not object.is_complete %}
+            {% trans "looks incomplete;" %}
+            {% trans "these fields should be completed:" %}<br/>
+            {{ completeness_fields|join:', ' }}
```
Comment (submarcos) [inline geotrek/common/templates/common/publishable_completeness_fragment.html:15]: I think you should used translated names for field, not internal names

**Review by Chatewgne — APPROVED** [review]

File: geotrek/common/templates/common/publishable_completeness_fragment.html (hunk 2 above)
Comment (numahell) [reply geotrek/common/templates/common/publishable_completeness_fragment.html:15]: Now the view send verbose name instead of internal names.

**Review by submarcos — APPROVED** [review]

**Pull request merged**